apps/test/unit/code-studio/components/SublevelCardTest.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {assert} from '../../../util/reconfiguredChai';
+import SublevelCard from '@cdo/apps/code-studio/components/SublevelCard';
+
+const DEFAULT_PROPS = {
+  isLessonExtra: false,
+  sublevel: {
+    id: 1,
+    display_name: 'Choice 1',
+    description: 'Sublevel 1 is lots of fun',
+    thumbnail_url: 'some-fake.url/kittens.png',
+    url: '/s/script/stage/1/puzzle/2/sublevel/1',
+    position: 1,
+    letter: 'a',
+    perfect: false,
+    status: 'perfect'
```

Comment:
I'm a little confused about why perfect could be false, but status is 'perfect'?

Reply:
lol i can't. Thats just bad test data. I will update it. Thanks for catching